README.md

```diff
@@ -73,6 +84,16 @@ Supported options:
 
 Creates a hosted transaction within Berbix to initialize the client SDK. This works the same as `createTransaction()` except that the object returned includes an explicit `hostedUrl` property for hosted transactions.
 
+##### `createApiOnlyTransaction(options: object): Tokens`
```

Comment:
Do we want to add support in CreateTransactionOptions for the selfie consent?

Reply:
I think we should, but going to kick that to another PR for the sake of getting something usable released.